src/fivetran/lambdaOpsGenie/lambdaHandler.ts

```diff
@@ -31,16 +48,17 @@ async function update(state: string) {
         primary_key: ['id']
       }
     },
-    hasMore: false
+    hasMore: hasMore
   }
 }
-async function apiResponse(lastUpdate: string) {
+async function apiResponse(lastUpdate: string | undefined, paginationCounter: number) {
   if (typeof lastUpdate === 'undefined') {
```

Comment:
```suggestion
  if (lastUpdate === undefined) {
```

you shouldn't need the type check.